docs/deploying.adoc

```diff
@@ -237,6 +237,22 @@ virtualClusters:
 <2> File system location of a file containing the trust store's password.
 <3> (Optional) Trust store type. Supported types are: `PKCS12`, `JKS` and `PEM`.  Defaults to Java default key store type (PKCS12).
 
+The following illustrates connection to physical cluster using client credentials over SSL.
```

Comment:
"client credentials" is ambiguous. After all, when using SSL client credentials flow over the wire too. How would you feel about?
```suggestion
The following illustrates connection to physical cluster using TLS client authentication (aka Mutual TLS).
```